packages/ui/src/Link.module.scss

```diff
@@ -1,5 +1,10 @@
 @use '../styles/constants' as c;
 
+button.buttonReset {
+  border: none;
+  background: none;
+}
+
 @mixin link {
```

Comment:
I just saw we use `outline: none;` for `:focus`. Shoud it be `outline: transparent solid c.$outlineWidth` instead?